src/xo-app/pool/tab-logs.js

```diff
@@ -1,19 +1,55 @@
 import _ from 'intl'
-import ActionRow from 'action-row-button'
 import React, { Component } from 'react'
-import TabButton from 'tab-button'
+import SortedTable from 'sorted-table'
 import { deleteMessage } from 'xo'
 import { createPager, createSelector } from 'selectors'
```

Comment:
You shouldn't need `createPager` anymore. Make sure you removed everything from the old pagination system as `SortedTable` handles it.